libres/lib/include/ert/enkf/hook_manager.hpp

```diff
@@ -55,4 +56,24 @@ hook_manager_iget_hook_workflow(const hook_manager_type *hook_manager,
                                 int index);
 extern "C" int hook_manager_get_size(const hook_manager_type *hook_manager);
 
+/** Runpath contains the data for one line in the runpath list file */
+class Runpath {
+public:
+    explicit Runpath(int iens, int iter, std::string runpath,
+                     std::string jobname);
+    explicit Runpath(int iens, int iter, const char *runpath,
+                     const char *jobname);
+    /** The index of the run in the ensemble */
+    int iens;
+    /** The index of the iteration */
+    int iter;
+    /** Absolute path of the runs directory */
+    std::string runpath;
```

Comment:
If this is a path, use `std::filesystem::path`

Reply:
It is a path, but we are never manipulating it as a path so it would make sense to just keep it as a string and change it once the use site starts using it as a path.